inc/usersim/ke.h

```diff
@@ -173,6 +174,38 @@ usersim_free_semaphores();
 
 #pragma endregion semaphores
 
+#pragma region events
+
+typedef enum _EVENT_TYPE
+{
+    NotificationEvent,
+    SynchronizationEvent
+} EVENT_TYPE;
+
+typedef struct _kevent
+{
+    usersim_object_type_t object_type;
+    EVENT_TYPE type;
+    KSPIN_LOCK spin_lock;
+    LONG signalled;
```

Comment:
```suggestion
    LONG signaled;
```
Two L is British spelling, one L is American spelling.  Elsewhere we use American spellings.